### PR TITLE
hypershift: Fixup things

### PR DESCRIPTION
**Pull request by sjenning**
PR contains 3 commits
* revert multiarch disable now that https://issues.redhat.com/browse/ART-5969 is merged
* fixups after https://github.com/openshift/release/pull/37630 fyi @stbenjam 
* rebased commit from https://github.com/openshift/release/pull/36324 to avoid image builds on 4.11 conformance

### CODE REVIEW

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml [hunk 1]
```diff
@@ -131,6 +131,14 @@ tests:
     env:
       CNV_PRERELEASE_VERSION: "4.14"
     workflow: hypershift-kubevirt-conformance
+- as: e2e-kubevirt-mce-conformance
```
Comment (sjenning) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml:134]: syncing this test from 4.13.  somehow it got dropped.

Comment (davidvossel) [reply ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml:134]: yep. we'll update some of the env vars later

**Pull request merged**